protocol/v2/ssv/runner/runner.go

```diff
@@ -224,15 +224,14 @@ func (b *BaseRunner) basePartialSigMsgProcessing(
 
 // didDecideCorrectly returns true if the expected consensus instance decided correctly
 func (b *BaseRunner) didDecideCorrectly(prevDecided bool, decidedMsg *specqbft.SignedMessage) (bool, error) {
-	decided := decidedMsg != nil
-	decidedRunningInstance := decided && b.State.RunningInstance != nil && decidedMsg.Message.Height == b.State.RunningInstance.GetHeight()
-
-	if !decided {
+	if decidedMsg == nil {
```

Comment:
This change is from another PR, https://github.com/bloxapp/ssv-spec/pull/342, right? I think it's worth mentioning in the PR description as well.